helpers/focus.js

```diff
@@ -14,6 +14,31 @@ const focusableElements = {
 	textarea: true
 };
 
+export function getAllFocusableDescendants(node, includeHidden = true) {
```

Comment:
Could you add this file to the helper readme. I noticed it wasn't there and since you added to it you get more work as a prize!

Reply:
Oh! Good point